pkg/engine/lifecycle_test.go

```diff
@@ -1535,14 +1535,14 @@ func TestRefreshDeleteDependencies(t *testing.T) {
 		deploytest.NewProviderLoader("pkgA", semver.MustParse("1.0.0"), func() (plugin.Provider, error) {
 			return &deploytest.Provider{
 				ReadF: func(urn resource.URN, id resource.ID,
-					state resource.PropertyMap) (resource.PropertyMap, resource.Status, error) {
+					state resource.PropertyMap) (plugin.ReadResult, resource.Status, error) {
 
 					switch id {
 					case "0", "4":
 						// We want to delete resources A::0 and A::4.
-						return nil, resource.StatusOK, nil
+						return plugin.ReadResult{}, resource.StatusOK, nil
 					default:
-						return state, resource.StatusOK, nil
+						return plugin.ReadResult{Outputs: state}, resource.StatusOK, nil
```

Comment:
Is this deliberately not returning inputs?

Reply:
Yes--it's not necessary for this test.